<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CallablePropertiesBase</class>
 <widget class="QDialog" name="CallablePropertiesBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Callable Properties</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Python return type</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="pyType"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Python arguments</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="pyArgs"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Annotations</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="abortOnExcCb">
        <property name="text">
         <string>AbortOnException</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="noTypeHintCb">
        <property name="text">
         <string>NoTypeHint</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="allowNoneCb">
        <property name="text">
         <string>AllowNone</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="numericCb">
        <property name="text">
         <string>Numeric</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>API</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="api"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>AutoGen</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="autoGen"/>
        </item>
       </layout>
      </item>
      <item row="2" column="1" rowspan="4">
       <layout class="QFormLayout" name="formLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>PostHook</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="postHook"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>PreHook</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="preHook"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>PyName</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="pyName"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>PyQtSignalHack</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="signalHack"/>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="defaultCb">
        <property name="text">
         <string>Default</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="disallowNoneCb">
        <property name="text">
         <string>DisallowNone</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" rowspan="2">
       <layout class="QFormLayout" name="formLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>DocType</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="docType"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Encoding</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="encodingCb"/>
        </item>
       </layout>
      </item>
      <item row="6" column="1">
       <widget class="QCheckBox" name="pyIntCb">
        <property name="text">
         <string>PyInt</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QCheckBox" name="factoryCb">
        <property name="text">
         <string>Factory</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QCheckBox" name="releaseGILCb">
        <property name="text">
         <string>ReleaseGIL</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QCheckBox" name="finalCb">
        <property name="text">
         <string>final</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QCheckBox" name="transferCb">
        <property name="text">
         <string>Transfer</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QCheckBox" name="holdGILCb">
        <property name="text">
         <string>HoldGIL</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QCheckBox" name="transferBackCb">
        <property name="text">
         <string>TransferBack</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QCheckBox" name="keepRefCb">
        <property name="text">
         <string>KeepReference</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QCheckBox" name="transferThisCb">
        <property name="text">
         <string>TransferThis</string>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QCheckBox" name="newThreadCb">
        <property name="text">
         <string>NewThread</string>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <layout class="QFormLayout" name="formLayout_5">
        <item row="0" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>TypeHint</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="typeHint"/>
        </item>
       </layout>
      </item>
      <item row="12" column="0">
       <widget class="QCheckBox" name="noArgParserCb">
        <property name="text">
         <string>NoArgParser</string>
        </property>
       </widget>
      </item>
      <item row="12" column="1">
       <widget class="QCheckBox" name="lenCb">
        <property name="text">
         <string>__len__</string>
        </property>
       </widget>
      </item>
      <item row="13" column="0">
       <widget class="QCheckBox" name="noCopyCb">
        <property name="text">
         <string>NoCopy</string>
        </property>
       </widget>
      </item>
      <item row="13" column="1">
       <widget class="QCheckBox" name="matmulCb">
        <property name="text">
         <string>__matmul__</string>
        </property>
       </widget>
      </item>
      <item row="14" column="0">
       <widget class="QCheckBox" name="noDerivedCb">
        <property name="text">
         <string>NoDerived</string>
        </property>
       </widget>
      </item>
      <item row="14" column="1">
       <widget class="QCheckBox" name="imatmulCb">
        <property name="text">
         <string>__imatmul__</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>14</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pyType</tabstop>
  <tabstop>pyArgs</tabstop>
  <tabstop>abortOnExcCb</tabstop>
  <tabstop>allowNoneCb</tabstop>
  <tabstop>api</tabstop>
  <tabstop>autoGen</tabstop>
  <tabstop>defaultCb</tabstop>
  <tabstop>disallowNoneCb</tabstop>
  <tabstop>docType</tabstop>
  <tabstop>encodingCb</tabstop>
  <tabstop>factoryCb</tabstop>
  <tabstop>finalCb</tabstop>
  <tabstop>holdGILCb</tabstop>
  <tabstop>keepRefCb</tabstop>
  <tabstop>newThreadCb</tabstop>
  <tabstop>noArgParserCb</tabstop>
  <tabstop>noCopyCb</tabstop>
  <tabstop>noDerivedCb</tabstop>
  <tabstop>noTypeHintCb</tabstop>
  <tabstop>numericCb</tabstop>
  <tabstop>postHook</tabstop>
  <tabstop>preHook</tabstop>
  <tabstop>pyName</tabstop>
  <tabstop>signalHack</tabstop>
  <tabstop>pyIntCb</tabstop>
  <tabstop>releaseGILCb</tabstop>
  <tabstop>transferCb</tabstop>
  <tabstop>transferBackCb</tabstop>
  <tabstop>transferThisCb</tabstop>
  <tabstop>typeHint</tabstop>
  <tabstop>lenCb</tabstop>
  <tabstop>matmulCb</tabstop>
  <tabstop>imatmulCb</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CallablePropertiesBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CallablePropertiesBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
